### PR TITLE
Added handling for ties

### PR DESCRIPTION
**Pull request by falkirks**
Takes an annoying amount of code to add commas and "and" to the output. Maybe there is a better way...

See #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,13 +62,36 @@ fetchCats()
...
+
+      let pos = winnerDesc.lastIndexOf(',');
+      winnerDesc = winnerDesc.substring(0, pos) + ' and' + winnerDesc.substring(pos+1);
+    }
+    console.log(`The fattest ${winnerDesc} ${fattestCats[0].lbs} lbs and ${fattestCats[0].oz} oz.`.output);
     setTimeout(() => console.log("Opening cat profile..."), 2000);
-    setTimeout(() => opener(fattestCat.url), 4000);
+    setTimeout(() => {
+        for(let fattestCat of fattestCats){
+            opener(fattestCat.url);
```
Comment (sbilstein) [inline index.js:97]: is this vulnerable to a malicious SPCA volunteer uploading hundreds of cats with the same weight of 666lbs? A user's browser could have 2 many tabs. Please coordinate with the security team for testing.

Comment (lexiross) [reply index.js:97]: hahahah hey @sbilstein!

v good point - we should slice it to a reasonable number, let's say 10 fattest cats, just in case

**Comment by lexiross** [comment]
This looks great @Falkirks - can you just add the slice per my comment above? And if you can think of a more elegant way to construct the message, go for it, but otherwise no worries :)

**Comment by falkirks** [comment]
@lexiross Ok. Added that.

**Comment by lexiross** [comment]
Awesome, thank you so much @Falkirks 🐱

**Pull request merged**